static/app/views/replays/detail/focusArea.tsx

```diff
@@ -13,9 +13,7 @@ import FocusTabs from './focusTabs';
 import MemoryChart from './memoryChart';
 
 type Props = {
-  event: Event;
-  eventWithSpans: Event | undefined;
-  memorySpans: MemorySpanType[] | undefined;
+  replay: Replay;
```

Comment:
I feel like the term `replay` is a bit vague and overloaded. maybe `replayData`? -- no strong opinions

Reply:
Renamed in the 1st PR to be `ReplayReader`